release.sh

```diff
@@ -25,28 +29,54 @@ changelog() {
 }
 
 bump() {
-  # Bump package version and generate changelog
-  VERSION="${NEXT/v/}"
 
-  echo "Bump version to ${VERSION}"
+  if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
+      printfln "Exiting for pull request without versioning."
+  fi
+
+  last_tag=$(git tag --sort=-creatordate | head -n 1)
+
+  printfln "$last_tag"
+
+  if [ "$BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+      echo "Bumping the version: ${last_tag} -> ${NEXT}"
+      git tag "${NEXT}"
+
+      hub release create "$NEXT" -m "$NEXT" || true
+  fi
+}
+
+compare_and_release() {
+  ## Compare the package.json file from two recent commits to master branch and export value to NEXT variable
+  ## if differs else NEXT=false
 
-  # Update version in the following files
-  sed -i "s/\(\"version\":\s*\"\)[^\"]*\(\"\)/\1${VERSION}\2/g" package.json
+  previous_commit_hash=$(git rev-parse @~)
```

Comment:
Isn't this the same? We are trying to get the last commit in master?

```suggestion
  previous_commit_hash=$(git rev-parse HEAD)
```

Reply:
Not last but second last. @ -> last @~ -> second last. when PR is merged lst commit contains changed version

Reply:
Cool. Can you add a comment here that says.

```
# Get the second last commit from the master branch after merge.
```